sdks/python/apache_beam/io/gcp/datastore/v1/datastoreio_test.py

```diff
@@ -191,6 +192,26 @@ def check_DatastoreWriteFn(self, num_entities):
       self.assertEqual((num_entities - 1) / _Mutate._WRITE_BATCH_SIZE + 1,
                        self._mock_datastore.commit.call_count)
 
+  def test_DatastoreWriteLargeEntities(self):
+    """60*100kB entities gets split over two Commit RPCs."""
+    with patch.object(helper, 'get_datastore',
+                      return_value=self._mock_datastore):
+      entities = [e.entity for e in
+                  fake_datastore.create_entities(60)]
+
+      datastore_write_fn = _Mutate.DatastoreWriteFn(self._PROJECT)
+      datastore_write_fn.start_bundle()
+      for entity in entities:
+        datastore_helper.add_properties(
+            entity, {'large': u'A' * 100000}, exclude_from_indexes=True)
+        datastore_write_fn.process(WriteToDatastore.to_upsert_mutation(entity))
+      datastore_write_fn.finish_bundle()
+
+      self.assertEqual(
+          math.ceil(sum(e.ByteSize() for e in entities) /
```

Comment:
Actually that makes more sense - if someone increases the size per RPC later, the test might end up testing for 1 RPC, which would miss the point of the test.